encoder/generator.go

```diff
@@ -3,25 +3,27 @@ package encoder
 import (
 	"bytes"
 	"fmt"
-	"github.com/iancoleman/strcase"
 	gast "go/ast"
 	"math/bits"
 	"os"
 	"sort"
 	"strings"
 	"text/template"
 
+	"github.com/iancoleman/strcase"
+
 	"cuelang.org/go/cue"
 	"cuelang.org/go/cue/ast"
 	"cuelang.org/go/cue/errors"
 )
 
-const attrname = "cuetsy"
 const (
-	attrTarget      = "targetType"
+	attrname        = "cuetsy"
 	attrEnumDefault = "enumDefault"
 )
 
+var attrTarget = [...]string{"kind", "targetType"}
```

Comment:
Meh. i know i said we could do this, but actually looking at the implementation, it seems silly to try to attain backwards compatibility here. This is an easy fix in the only place we know cuetsy to really exist - grafana/grafana.

So let's only support `"kind"`

Reply:
I was thinking that we might have to update all cue files there :), might be painful for now. ok let's keep it simple.